### PR TITLE
Fix and re-enable RLlib example test

### PR DESCRIPTION
**Pull request by JVillella**
cc @Adaickalavan 

Some minor typos in the rllib.py file and re-enabled the associated test. Please try this branch on your machine.

### CODE REVIEW

File: tests/test_examples.py [hunk 1]
```diff
@@ -24,20 +24,19 @@ def test_multi_instance_example():
     )
 
 
-# TODO: "rllib example" is causing "Abort trap: 6", fix later
```
Comment (liamchzh) [inline tests/test_examples.py:27]: Is this still an issue? Otherwise, LGTM

**Review by liamchzh — APPROVED** [review]

File: examples/rllib.py [hunk 2]
```diff
@@ -205,7 +205,7 @@ def main(
     save_model_path = str(Path(__file__).expanduser().resolve().parent / "model")
     parser.add_argument(
         "--save_model_path",
-        type=int,
+        type=str,
```
Comment (Gamenot) [inline examples/rllib.py:208]: This is an odd looking bug.

**Review by Gamenot — APPROVED** [review]

**Review by Adaickalavan — APPROVED** [review]

File: tests/test_examples.py (hunk 1 above)
Comment (JVillella) [reply tests/test_examples.py:27]: No, that's why it was re-enabled.

File: examples/rllib.py (hunk 2 above)
Comment (JVillella) [reply examples/rllib.py:208]: Yea...

**Pull request merged**